src/UNIV2LPOracle.t.sol

```diff
@@ -821,7 +851,31 @@ contract UNIV2LPOracleTest is DSTest {
         assertEq(daiEthLPOracle.zzz(), 0);
     }
 
-    function test_stop_start_poke() public {
+    function test_void_start_poke() public {
+        daiEthLPOracle.poke();                                      // Poke DAI-ETH LP Oracle
+        daiEthLPOracle.kiss(address(this));                         // Whitelist caller on DAI-ETH LP Oracle
+        (bytes32 val, bool has) = daiEthLPOracle.peep();            // Query queued price of DAI-ETH LP Oracle
+        uint256 resVal = uint256(val);                              // Cast queued price as uint256
+
+        assertTrue(resVal < 100 ether && resVal > 50 ether);        // 57327394135985707908 at time of test
+        assertTrue(has);                                            // Verify Oracle has valid value
+
+        daiEthLPOracle.void();
+        // No time change between void and start
+
+        daiEthLPOracle.start();
+        assertEq(uint256(daiEthLPOracle.stopped()), 0);
+
+        daiEthLPOracle.poke();
+
+        (val, has) = daiEthLPOracle.peep();                         // Query queued price of DAI-ETH LP Oracle
+        resVal = uint256(val);                                      // Cast queued price as uint256
+
+        assertTrue(resVal < 100 ether && resVal > 50 ether);        // 57327394135985707908 at time of test
+        assertTrue(has);                                            // Verify Oracle has valid value
+    }
+
+    function test_stop_void_start_poke() public {
```

Comment:
I think we might be able to remove this test as we `test_void_start_poke` and `test_stop_start`.